prow/config/tide_test.go

```diff
@@ -148,6 +148,83 @@ func TestMergeMethod(t *testing.T) {
 		}
 	}
 }
+func TestMergeTemplate(t *testing.T) {
+	ti := &Tide{
+		MergeTemplate: map[string]TideMergeCommitTemplate{
+			"kubernetes/kops": TideMergeCommitTemplate{
+				TitleTemplate: "",
+				BodyTemplate:  "",
+			},
+			"kubernetes/charts": TideMergeCommitTemplate{
+				TitleTemplate: "{{ .Number }}",
+				BodyTemplate:  "",
+			},
+			"helm/charts": TideMergeCommitTemplate{
+				TitleTemplate: "",
+				BodyTemplate:  "{{ .Body }}",
+			},
+			"kubernetes-helm": TideMergeCommitTemplate{
+				TitleTemplate: "{{ .Title }}",
+				BodyTemplate:  "{{ .Body }}",
+			},
+		},
+	}
+
+	var testcases = []struct {
+		org      string
+		repo     string
+		expected TideMergeCommitTemplate
+	}{
+		{
+			"kubernetes",
+			"kubernetes",
+			TideMergeCommitTemplate{
```

Comment:
The intent was :-1: 

```go
{
    org: "kubernetes",
    repo: "kubernetes",
    expected: ...
}
```

If a field is omitted it is not necessary to name it and set it to `nil` but if if a field is set we want to use literal names for the fields so ordering is irrelevant and refactoring it is easier in the future. @cjwagner weird that `lint` didn't catch this with `go vet`

Reply:
Gotcha, I was following the structure from the test above it.